src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -124,7 +126,7 @@ Amber::Server.configure do |settings|
   # Log: Is the logger that will be used for Amber and it defaults to ::Logger.new(STDOUT).
   # You can supply a custom logger.
   #
-  # settings.logger = ::Logger.new(STDOUT)
+  # settings.logger = Amber::Environment::Logger.new(File.open("<%= @name %>.log", "w"))
```

Comment:
The comment above probably doesn't need to say `and it defaults to ::Logger.new(STDOUT)`. How about something that's less likely to fall out of date as things change:

> settings.logger is the logger that Amber and other capable shards in the project will use instead of writing directly to STDOUT. Supply a custom logger to write to syslog, etc.

Reply:
Done! 👍